kratos/processes/sub_model_part_skin_detection_process.h

```diff
@@ -60,6 +60,19 @@ bool IsSelected(const Geometry<Node<3>>::PointsArrayType& rNodes) const override
 
 };
 
+// Select faces where almost one node does not belong to given SubModelPart.
+class SelectIfOneNodeNotOnSubModelPart: public FaceSelector
+{
+std::vector<std::string> mNames;
```

Comment:
Add some more detailed description Doxygen and tabulate the interior of the class

Reply:
I've added some tabs and doxygen to the existing code.

Reply:
> tabulate the interior of the class

Do I continue tabulating the entire existing class?